src/python/pants/option/global_options.py

```diff
@@ -50,6 +51,7 @@ class ExecutionOptions(datatype([
   @classmethod
   def from_bootstrap_options(cls, bootstrap_options):
     return cls(
+      enable_remoting=bootstrap_options.enable_remoting,
```

Comment:
The `--enable-pantsd` flag isn't the best example to follow from a naming perspective:
1. it's not namespaced
2. it uses "enable" in the name, which is redundant

So maybe: `--[no-]remote-execution`.

Reply:
Good suggestion. Thanks!